src/plugins/csp_configuration_provider/server/csp_handlers.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CoreSetup, OnPreResponseHandler, OpenSearchClient } from '../../../core/server';
+import { CspClient } from './types';
+
+const OPENSEARCH_DASHBOARDS_CONFIG_INDEX_NAME = '.opensearch_dashboards_config';
```

Comment:
to verify, the ideal of putting it into a new config was to mitigate the current issue of private tenancy and global tenancy? would this be too generic of a name since it seems it can house more than just csp rules? perhaps we can make this configurable to avoid any potential conflicts?

like: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/server/opensearch_dashboards_config.ts#L50

by default it will defined as whatever we decide.

Reply:
1 Yes. The key motivation is to avoid the issue of private/global tenancies. 
2 It is intended to have a generic name for housing more configurations which have similar issue.
3 Great suggestion! I just made a change to make the name configurable